cmd/openshift-install/create.go

```diff
@@ -186,6 +193,49 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 	}
 }
 
+// appendRouterCAToClusterCA appends router CA to cluster CA in kubeconfig
+func appendRouterCAToClusterCA(config *rest.Config, directory string) (err error) {
+	client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return errors.Wrap(err, "creating a Kubernetes client")
+	}
+
+	routerSecret, err := client.CoreV1().Secrets("openshift-ingress-operator").Get("router-ca", metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
```

Comment:
Do we really want to silently ignore a missing `router-ca` secret?

Reply:
I will check, but I _think_ there's a possible expectation that it won't be created.

Reply:
The secret will always be created.  However, I'd suggest using the `router-ca` _configmap_ in the `openshift-config-managed` namespace, which will be created when some clusteringress is using the operator-generated default certificate generated using that CA certificate, and will be absent otherwise.

Reply:
but also, at the very least log the error. preferably don't swallow it at all.

Reply:
there's an issue where the configMap isn't getting created (being discussed in slack atm), so until that is resolved, I'll fetch the secret (added a TODO note, or maybe i should file an issue instead?)

I've added a log msg for that error

Reply:
Won't we want to poll/wait until the config-map has a non-empty CA instead of giving up?